build/pause/Dockerfile

```diff
@@ -15,6 +15,6 @@
 ARG BASE
 FROM ${BASE}
 ARG ARCH
-ADD bin/pause-linux-${ARCH} /pause
 USER 65535:65535
-ENTRYPOINT ["/pause"]
+COPY --chown=65535:65535 --chmod=550 bin/pause-linux-${ARCH} /pause
```

Comment:
I'm actually torn on whether we should chmod 555, so happy to hear others thoughts on this. At the very least we should do a chown to set user and group ownership to the user and group we run the container as.